stable/kommander/values.yaml

```diff
@@ -97,6 +97,10 @@ kommander-thanos:
   enabled: true
   thanosAddress: "prometheus-kubeaddons-prom-prometheus.kubeaddons.svc.cluster.local:10901"
 
+kommander-karma:
+  enabled: true
```

Comment:
I am not sure if we want this enabled by default. I suggest you set it to false in kubeaddons-configs.

Reply:
@branden ^

Reply:
My understanding is that we want Karma enabled by default when Kommander is deployed.

Reply:
/cc @gracedo @philipnrmn

Reply:
That's also my understanding - that Karma should be enabled by default